LiveSplit.AutoSplitters.xml

```diff
@@ -10770,4 +10770,26 @@
         <Description>Autosplitting available (by LurkingSR)</Description>
 	<Website>https://github.com/samehhamadeh/AutoSplitters</Website>
     </AutoSplitter>
+	<AutoSplitter>
+        <Games>
+            <Game>The Lion King (SNES)</Game>
+        </Games>
+        <URLs>
+            <URL>https://github.com/kannadan/autosplitter/blob/master/autosplit_lion_king_snes.asl</URL>
```

Comment:
Use the raw link

Reply:
Funny, thought I had but must have switched them around by accident. Added a fix commit